daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/unparsers/runtime1/SpecifiedLengthUnparsers.scala

```diff
@@ -87,122 +86,31 @@ final class SpecifiedLengthExplicitImplicitUnparser(
           // but we don't know if the length units characters or bits/bytes.
           lengthUnits match {
             case LengthUnits.Bits | LengthUnits.Bytes =>
-              unparseVarWidthCharactersInBits(state)
-            case LengthUnits.Characters =>
-              unparseVarWidthCharactersInCharacters(state)
+              eUnparser.unparse1(state)
+            case LengthUnits.Characters => {
+              //
+              // variable-width encodings and lengthUnits characters, and lengthKind explicit
+              // is not supported (currently) for complex types
+              //
+              state.schemaDefinitionUnless(
```

Comment:
Just trying to remember, is it still an SDE if it is something that just isn't supported yet?

Reply:
Good point, I think we usually use `subset`/`subsetError`, which is an SDE but mentions it's a "subset".